go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -1014,6 +1016,7 @@ var defaultConfig = TabletConfig{
 	Consolidator:                Enable,
 	ConsolidatorStreamTotalSize: 128 * 1024 * 1024,
 	ConsolidatorStreamQuerySize: 2 * 1024 * 1024,
+	ConsolidatorQueryWaiterCap:  0,
```

Comment:
nit: this can be ignored to be kept in default config for `0` as default

Reply:
I dropped this line and use 0 as default in the flag.